components/Navefooter.vue

```diff
@@ -44,7 +49,23 @@ import Vue from 'vue'
 import VueTypedJs from 'vue-typed-js'
 Vue.use(VueTypedJs)
 
+const socials = [
```

Comment:
n existe essa variavel fora do componete?

Reply:
não, só vai aparecer aqui, se for o caso de usar em outro lugar seria interessante colocar global